en-US/mozorg/internet-health/digital-inclusion.lang

```diff
@@ -0,0 +1,265 @@
+## internet_health_open_innovation ##
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/digital-inclusion/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/digital-inclusion/
+
+
+;Digital Inclusion: Vital to Internet Health
+Digital Inclusion: Vital to Internet Health
+
+
+;Digital inclusion means promoting diversity, practicing respect, and supporting universal access online.
+Digital inclusion means promoting diversity, practicing respect, and supporting universal access online.
+
+
+;Privacy &amp; Security
+Privacy &amp; Security
+
+
+;Digital Inclusion
+Digital Inclusion
+
+
+## TAG: internet_health_open_innovation
```

Comment:
Remove this tag, there are no tags for the menu in this template
https://github.com/alexgibson/bedrock/blob/06448a41063b2d870df9f018324b8db2ccd3326a/bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html#L24-L31